src/forms/EditListingContactForm/EditListingContactForm.js

```diff
@@ -34,38 +34,24 @@ export const EditListingPaymentFormComponent = (props) => {
         const submitDisabled = invalid || disabled || submitInProgress
         const { updateListingError, showListingsError } = fetchErrors || {}
 
-        const questionATitle = intl.formatMessage({ id: 'EditListingPaymentForm.questionATitle' })
-        const questionATitlePlaceholderMessage = intl.formatMessage({
-          id: 'EditListingPaymentForm.questionATitlePlaceholder',
-        })
-        const questionBTitle = intl.formatMessage({ id: 'EditListingPaymentForm.questionBTitle' })
+        const questionBTitle = intl.formatMessage({ id: 'EditListingContactForm.questionBTitle' })
         const questionBTitlePlaceholderMessage = intl.formatMessage({
-          id: 'EditListingPaymentForm.questionBTitlePlaceholder',
+          id: 'EditListingContactForm.questionBTitlePlaceholder',
         })
 
         return (
           <Form onSubmit={handleSubmit} className={classes}>
             {updateListingError ? (
               <p className={css.error}>
-                <FormattedMessage id="EditListingPaymentForm.updateFailed"/>
+                <FormattedMessage id="EditListingContactForm.updateFailed"/>
               </p>
             ) : null}
             {showListingsError ? (
               <p className={css.error}>
-                <FormattedMessage id="EditListingPaymentForm.showListingFailed"/>
+                <FormattedMessage id="EditListingContactForm.showListingFailed"/>
               </p>
             ) : null}
 
-            <FieldTextInput
-              id="payPalEmail"
-              name="payPalEmail"
-              className={css.title}
-              type="text"
-              label={questionATitle}
-              placeholder={questionATitlePlaceholderMessage}
-              autoFocus
-            />
-
             <FieldTextInput
```

Comment:
Can we change this to type `tel` or whatever it is? https://www.w3schools.com/tags/att_input_type_tel.asp. A pattern to validate would be great too.

Reply:
I have the next PR that addresses this, you can't add type="tel" to this component, however, Flex gives you a Text Phone component that does what you are asking for.